config/config_params_test.go

```diff
@@ -155,12 +155,30 @@ var _ = Describe("Config override empty", func() {
 			Expect(cp.BPFEnabled).To(BeFalse())
 
 			By("Ignoring a lower-priority config update")
-			changed, err = cp.UpdateFrom(map[string]string{"BPFEnabled": "true"}, EnvironmentVariable)
+			// Env vars get converted to lower-case before calling UpdateFrom.
+			changed, err = cp.UpdateFrom(map[string]string{"bpfenabled": "true"}, EnvironmentVariable)
```

Comment:
I see other test places that are still passing a mixed case name here. Should they be updated too?

(dataplane/linux/int_dataplane_test.go, daemon/daemon_test.go, and other places in this file)

Reply:
OK, but those other cases that I noted are also using `EnvironmentVariable`.

Reply:
Ah, I see what you mean; would be more realistic but I think the bases are covered now so would rather avoid another respin on this PR.